datadog_checks_dev/pyproject.toml

```diff
@@ -0,0 +1,117 @@
+[build-system]
+requires = [
+    "hatchling>=0.11.2",
+    "setuptools; python_version < '3.0'",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "datadog-checks-dev"
+description = "The Datadog Checks Developer Tool"
+readme = "README.md"
+license = "BSD-3-Clause"
+keywords = [
+    "datadog",
+    "datadog agent",
+    "datadog check",
+    "dev tool",
+    "tests",
+]
+authors = [
+    { name = "Datadog", email = "packages@datadoghq.com" },
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "License :: OSI Approved :: BSD License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 2.7",
+    "Programming Language :: Python :: 3.8",
+]
+dependencies = [
+    "contextlib2; python_version < '3.0'",
+    "coverage>=5.0.3",
+    "flaky",
+    "mock",
+    "psutil",
+    "pytest",
+    "pytest-benchmark[histogram]>=3.2.1",
+    "pytest-cov>=2.6.1",
+    "pytest-mock",
+    "pyyaml==5.4.1",
+    "requests>=2.22.0",
+    "shutilwhich==1.1.0; python_version < '3.0'",
+    "six",
+    "subprocess32==3.5.4; python_version < '3.0'",
+    "tenacity",
+]
+dynamic = [
+    "version",
+]
+
+[project.optional-dependencies]
+cli = [
+    "aiohttp",
+    "aiomultiprocess",
+    "atomicwrites",
+    "beautifulsoup4>=4.9.3",
+    "build>=0.7.0",
+    "click~=8.0",
+    "codespell",
+    "colorama",
+    "datamodel-code-generator~=0.11.4",
+    "docker-compose>=1.25",
+    "in-toto>=0.4.2",
+    "jsonschema",
+    "markdown",
+    "orjson",
+    "packaging",
+    "pip-tools",
+    "platformdirs>=2.0.0a3",
+    "pyperclip>=1.7.0",
+    "pysmi",
```

Comment:
should `pysmi` still be pinned `>=0.3.4`?

Reply:
ty! artifact from https://github.com/DataDog/integrations-core/pull/11299